src/visualization/types/SingleStat/view.tsx

```diff
@@ -19,54 +24,91 @@ interface Props extends VisualizationProps {
 const SingleStat: FC<Props> = ({properties, result}) => {
   const {prefix, suffix, colors, decimalPlaces} = properties
 
-  return (
-    <LatestValueTransform table={result.table} allowString={true}>
-      {latestValue => {
-        const {
-          bgColor: backgroundColor,
-          textColor,
-        } = generateThresholdsListHexs({
-          colors,
-          lastValue: latestValue,
-          cellType: 'single-stat',
-        })
+  if (isFlagEnabled('useGiraffeGraphs')) {
+    const latestValues = getLatestValues(result.table)
```

Comment:
`getLatestValues` should come from Giraffe. We should export it and release another version of Giraffe and consume it here.

Reply:
Although not needed here, we should also take the opportunity to also export `formatStatValue`

Reply:
https://github.com/influxdata/giraffe/pull/502